src/pages/wallets/index.tsx

```diff
@@ -0,0 +1,524 @@
+import { Image } from "@/components/Image"
+import OldHeading from "@/components/OldHeading"
+import Text from "@/components/OldText"
+import PageHero from "@/components/PageHero"
+import PageMetadata from "@/components/PageMetadata"
+import { StandaloneQuizWidget } from "@/components/Quiz/QuizWidget"
+import { Simulator } from "@/components/Simulator"
+import { SIMULATOR_ID } from "@/components/Simulator/constants"
+import Translation from "@/components/Translation"
+
+import { getLastDeployDate } from "@/lib/utils/getLastDeployDate"
+import { getRequiredNamespacesForPage } from "@/lib/utils/translations"
+
+import { walletOnboardingSimData } from "@/data/WalletSimulatorData"
+
+import DappsImage from "@/public/doge-computer.png"
+import ETHImage from "@/public/eth-logo.png"
+import FindWalletImage from "@/public/wallets/find-wallet.png"
+import HeroImage from "@/public/wallets/wallet-hero.png"
+
+const Page = (props: BoxProps) => (
+  <Flex direction="column" align="center" width="full" m="0 auto" {...props} />
+)
+
+const GrayContainer = (props: BoxProps) => (
+  <Box
+    width="full"
+    py={16}
+    px={0}
+    mt={{ base: 4, lg: 8 }}
+    background="grayBackground"
+    boxShadow="inset 0px 1px 0px var(--eth-colors-tableItemBoxShadow)"
+    {...props}
+  />
+)
+
+const Content = (props: BoxProps) => <Box py={4} px={8} w="full" {...props} />
+
+const Intro = (props: ChildOnlyProp) => (
+  <Content pb={0} sx={{ h2: { mb: 0 } }} {...props} />
+)
+
+const TwoColumnContent = (props: BoxProps) => (
+  <Content
+    display="flex"
+    justifyContent="space-between"
+    p={8}
+    mb={12}
+    flexDirection={{ base: "column", lg: "row" }}
+    {...props}
+  />
+)
+
+const LeftColumn = (props: BoxProps) => (
+  <Box flex="0 1 50%" mr={{ base: 0, lg: 8 }} maxW="full" {...props} />
+)
+
+const RightColumn = (props: BoxProps) => (
+  <Box flex="0 1 50%" ml={{ lg: 8 }} maxW="full" {...props} />
+)
+
+const H2 = (props: ChildOnlyProp) => (
+  <OldHeading
+    fontSize={{ base: "2xl", md: "2rem" }}
+    lineHeight={1.4}
+    {...props}
+  />
+)
+
+const CardContainer = (props: BoxProps) => (
+  <Box
+    display={"flex"}
+    flexWrap={"wrap"}
+    marginLeft={-4}
+    marginRight={-4}
+    {...props}
+  />
+)
+
+export const StyledCard = (props: ComponentPropsWithRef<typeof Card>) => (
+  <Card
+    flex="1 1 30%"
+    minW="280px"
+    maxW={{ base: "full", md: "46%", lg: "31%" }}
+    m={4}
+    p={6}
+    {...props}
+  />
+)
+const ChecklistItem = (props: HorizontalCardProps) => (
+  <HorizontalCard
+    border={0}
+    display="flex"
+    alignItems="flex-start"
+    mb={4}
+    {...props}
+  />
+)
+const Divider = (props: BoxProps) => (
+  <Box my={16} w="10%" h="0.25rem" bgColor="homeDivider" {...props} />
+)
+
+const CalloutCardContainer = (props: ChildOnlyProp) => (
+  <CardContainer mt={16} {...props} />
+)
+
+const cards = [
+  {
+    emoji: ":dollar:",
+    title: <Translation id="page-wallets:page-wallets-manage-funds" />,
```

Comment:
There are multiple pages that have lists like this, and are located outside of the page component in most cases in the canonical/Gatsyb repo. 

I'm not sure exactly how we'd like to handle these, but I have been pulling these inside the components to enable the use of the `useTranslation` hook (`t("intl-key")`). If I understand correctly there are performance hits from using the `<Translation/>` component frequently, since each component has to mount and get the namespace information separately.

Not a blocker for this PR, but I was curious what others were thinking or preferred. @nhsz Am I correct that the `useTranslation` approach is generally more performant than the `Translation` component?

Reply:
Thanks for pointing this out and explaining @wackerow.  Refactored this to remove the use of the Translation component in these lists.

Reply:
Interesting topic to talk about in devs sync.

My 2 cents: the problem with using `t` for everything is the lack of support for `htmr` (internally `Translation` also uses the `t` function) and not using a standard approach that prevents potential bugs. Having a mixed usage of `t` & `Translation` is not ideal IMO. I'd prefer to always use the `Translation` component for any jsx rendering we need to do. This would prevent us from having to remember to update the render method when we add rich content to the translations. The `t` function should only be used for non jsx things.